src/client/src/components/SignUp.jsx

```diff
@@ -119,6 +128,21 @@ function SignUp() {
                     </label>
                     <br></br>
                     <br></br>
+                    <label className="PlaceholderSignup">Account Type
+                    <br />
+                    <Select required id="select" value={userData.role} 
+                    sx={{width: 250, height: 40, backgroundColor: "#a3c5d0",borderColor: "white"}}
+                    onChange={(e) =>
+                      setUserData({ ...userData, role: e.target.value })
+                    }>
+                      <MenuItem value={"User"}>User</MenuItem>
+                      <MenuItem value={"Recruiter"}>Recruiter</MenuItem>
+                      <MenuItem value={"Administrator"}>Administrator</MenuItem>
+                      <MenuItem value={"Illegal"}>Illegal</MenuItem>
```

Comment:
What is this selection used for ?

Reply:
It's there to test the collection constraint. I'll remove it.